src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1

```diff
@@ -1168,3 +1168,47 @@ function Test-SnapshotConfigDiskAccessNetworkPolicy
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Testing the new parameters 
+Tier
+LogicalSectorSize 
+in the New-AzDiskConfig cmdlet.  
+#>
+function Test-DiskConfigTierSectorSizeReadOnly
```

Comment:
This test will also have the New-AzDiskUpdateConfig changes with the ReadOnly parameters, hence the ReadOnly part of the name.